nlp/2021_06_29_long_range_transformers/LongRangeTransformers.ipynb

```diff
@@ -0,0 +1,1940 @@
+       "                            \n",
+       "var gd = document.getElementById('3f6e2e10-173d-41bd-9fe9-8f7adef1dec9');\n",
+       "var x = new MutationObserver(function (mutations, observer) {{\n",
+       "        var display = window.getComputedStyle(gd).display;\n",
+       "        if (!display || display === 'none') {{\n",
+       "            console.log([gd, 'removed!']);\n",
+       "            Plotly.purge(gd);\n",
+       "            observer.disconnect();\n",
+       "        }}\n",
+       "}});\n",
+       "\n",
+       "// Listen for the removal of the full notebook cells\n",
+       "var notebookContainer = gd.closest('#notebook-container');\n",
+       "if (notebookContainer) {{\n",
+       "    x.observe(notebookContainer, {childList: true});\n",
+       "}}\n",
+       "\n",
+       "// Listen for the clearing of the current output cell\n",
+       "var outputEl = gd.closest('.output');\n",
+       "if (outputEl) {{\n",
+       "    x.observe(outputEl, {childList: true});\n",
+       "}}\n",
+       "\n",
+       "                        })                };                });            </script>        </div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig = go.Figure()\n",
+    "colors=[px.colors.qualitative.Plotly[1],px.colors.qualitative.Plotly[2],px.colors.qualitative.Plotly[8]]\n",
+    "\n",
+    "for index, model_name in enumerate(result['model_name'].unique()):\n",
+    "    trace_data = result[result.model_name == model_name].sort_values('max_token_length').reset_index()\n",
+    "    color_map = list(Color(colors[index]).range_to(Color('#000'), 75))\n",
```

Comment:
`Color` class unknown at this stage.
As a general tip, it's always a good idea to run your notebook end-to-end before submitting it, to doublecheck that nothing is missing